src/main/java/bio/terra/workspace/service/job/JobService.java

```diff
@@ -69,28 +69,16 @@ public JobService(
       JobConfiguration jobConfig,
       IngressConfiguration ingressConfig,
       StairwayDatabaseConfiguration stairwayDatabaseConfiguration,
-      FlightBeanBag applicationContext,
       MdcHook mdcHook,
-      ObjectMapper objectMapper) {
+      StairwayComponent stairwayComponent,
+      FlightBeanBag flightBeanBag) {
     this.jobConfig = jobConfig;
     this.ingressConfig = ingressConfig;
     this.stairwayDatabaseConfiguration = stairwayDatabaseConfiguration;
     this.executor = Executors.newScheduledThreadPool(jobConfig.getMaxThreads());
     this.mdcHook = mdcHook;
-    StairwayExceptionSerializer serializer = new StairwayExceptionSerializer(objectMapper);
```

Comment:
then you lose StairwayExceptionSerializer. 
You may have to change method common-lib's initialize method. (maybe passing Stairway.Builder() if we are seeing too many parameters)?
@ddietterich wdyt?

Reply:
We definitely need a way to supply an exception serializer and override the Stairway built-in one.
If the param list gets too long, we can make an initializeStairway object with a builder and pass that in.

Reply:
Do we need this before merging this (and/or before the release)?

Reply:
The exception serializer improves the fidelity of exceptions caught with Stairway. But the default one in Stairway will deliver messages. So I vote not to let this hold up the merge. We can do it as a follow-on.

Let's discuss separately whether we can have one shared serializer for MC Terra based on the common exceptions or if we need to allow for individual ones per component.